src/js/components/Heading/StyledHeading.js

```diff
@@ -83,7 +84,7 @@ const StyledHeading = styled.h1`
   ${props => sizeStyle(props)}
   ${props => props.textAlign && textAlignStyle}
   ${props => props.truncate && truncateStyle}
-  ${props => props.colorProp && colorStyle}
+  ${props => (props.colorProp || props.theme.heading.color) && colorStyle}
```

Comment:
Think if we merge master into your branch, we'll get rid of some extraneous changes, not necessary but just making sure I'm catching the right things

Reply:
Yes, a lot of these are coming in from merging master into this branch, which have not been merged into the pagination branch. I will merge master into the pagination branch to make it more clear.

The files which had merge conflicts are what should be payed attention to here:
- Button.js
- StyledButtonKind.js
- DataTable.js